File: exercicios_java/exercicios_java/src/secao6/exercicio3.java
//Ler um número e verificar, se par ou ímpar. Se par, armazena em p,
//se ímpar em i.
package secao6;
import java.util.Scanner;

public class exercicio3 {
    public static void main(String[] args){
        int n, i, p;
        Scanner t = new Scanner(System.in);

        System.out.print("Digite um número: ");
        n = t.nextInt();

        if (n%2 == 0){
            p = n;
            System.out.print(p + " é par");
        }else {
            i = n;
            System.out.print(i + " é ímpar");
        }
        t.close();
    };
}

File: exercicios_py/secao6/exercicio3.py
#Ler um número e verificar, se par ou ímpar. 
# Se par, armazena em p, se ímpar em i.

n = int(input('Digite um número: '))
if n%2 == 0:
    p = n
    print("{} é par, n = p".format(n))
else:
    i = n
    print("{} é ímpar, n = i".format(n))